### ISSUE TITLE
FSE: Consider adding a sidebar template

### ISSUE THREAD

**Issue by melchoyce** [WordPress/twentytwentyone#175]
**Is your feature request related to a problem? Please describe.**
Sidebars are frequently requested features in themes, and we haven't thoroughly explored them in the context of a block-based, full-site-editing supported theme yet. Let's use Twenty Twenty-One as an example of how a block-based theme could explore a sidebar.

**Additional context**
This issue requires us to begin the block-based version of the theme before we can work on implementation.

![image](https://user-images.githubusercontent.com/2846578/94609895-6f7bb080-026d-11eb-9c4f-17bd4cdb9787.png)

**Comment by carolinan** [comment]
I don't think the FSE version should be in the same repository because the issues and pull requests risk getting mixed up.

**Comment by melchoyce** [comment]
Labeling out of scope and closing for now — we'll move over to the FSE repo whenever that gets created.
